lib/car.rb

```diff
@@ -0,0 +1,26 @@
+class Car
+
+  attr_accessor :make, :model, :monthly_payment, :loan_length, :color
+
+  def initialize(make_and_model, monthly_payment, loan_length)
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    make_and_model = {
+      make: @make,
+      model: @model
+      }
+    end
+
+
+  def color
+    []
+  end
+
+  def total_cost
+    54000
+  end
+
+  def paint!(paint_color)
+    color << paint_color
```

Comment:
Since our `color` started out as `nil`, we would want to re-assign `color` to the `paint_color` that you are passing this method.